http://bugs.gentoo.org/236654

make sure we return 0 when things work

patch by kouyu

--- bioapi-linux/apps/NonGUI_Sample/main.c
+++ bioapi-linux/apps/NonGUI_Sample/main.c
@@ -234,6 +234,7 @@
     }
     
     OutputToFile(userName, EnrolledTemplate);
+    return 0;
   }
 }
 
